rest/src/main/java/org/apache/unomi/rest/EventsCollectorEndpoint.java

```diff
@@ -75,13 +76,15 @@ public Response options() {
 
     @GET
     @Path("/eventcollector")
-    public EventCollectorResponse collectAsGet(EventsCollectorRequest eventsCollectorRequest, @QueryParam("timestamp") Long timestampAsString) {
+    public EventCollectorResponse collectAsGet(@NotNull EventsCollectorRequest eventsCollectorRequest,
+            @QueryParam("timestamp") Long timestampAsString) {
         return doEvent(eventsCollectorRequest, timestampAsString);
     }
 
     @POST
     @Path("/eventcollector")
-    public EventCollectorResponse collectAsPost(EventsCollectorRequest eventsCollectorRequest, @QueryParam("timestamp") Long timestampAsLong) {
+    public EventCollectorResponse collectAsPost(EventsCollectorRequest eventsCollectorRequest,
```

Comment:
Should be annotate also with NotNull for Post

Reply:
Also, if `eventsCollectorRequest` is set as `Not null` statement in [if](https://github.com/apache/unomi/pull/287/files#diff-6431c225a095ce6c2510037b49f2334e82807587b0d381f74cfd42a6e094aea6R97) can be removed

Reply:
Will do in separate PR